README.md

```diff
@@ -30,6 +30,8 @@ spec:
 
 ## Developing locally
 
+See the header of [`go.mod`](./go.mod) for the minimum supported version of Go.
```

Comment:
I would instead prefer to be consistent with the `go.mod`, would it fail with go `v1.19` if we update `go.mod` too?

Reply:
the go version inside `go.mod` is intended to be the minimum supported version ( [see](https://go.dev/doc/modules/gomod-ref) ).

So as the minimum supported version `go mod tidy` will fail it the version is not matched ( < `supported version`) 

```
faheddorgaa@MBP-de-fahed: go version
go version go1.19 darwin/amd64

faheddorgaa@MBP-de-fahed:$ go mod tidy
go: go.mod file indicates go 1.20, but maximum version supported by tidy is 1.19
```